src/views/workflow-history/workflow-history-events-card/workflow-history-events-card.types.ts

```diff
@@ -0,0 +1,6 @@
+import type { HistoryGroupEventMetadata } from '../workflow-history.types';
+
+export type Props = {
+  eventsMetadata: Pick<HistoryGroupEventMetadata, 'label' | 'status'>[];
+  showMissingEventPlaceholder?: boolean;
```

Comment:
nit: it'd be nice to have a shorter name here. 
```suggestion
  isEventMissing?: boolean;
```

Reply:
will go with `showEventPlaceholder` as it explicitly tells what happen if the event is missing